solvers/safe.py

```diff
@@ -0,0 +1,56 @@
+from benchopt import BaseSolver, safe_import_context
+
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from scipy import sparse
+    from slopescreening.solver.parameters import SlopeParameters
+    from slopescreening.solver.slope import slope_gp
+
+
+class Solver(BaseSolver):
+    name = "Safe"
+    stopping_strategy = "iteration"
+    install_cmd = "conda"
+    requirements = [
+        "pip:git+https://gitlab-research.centralesupelec.fr/2020elvirac/slope-screening@master"
+    ]
+    references = [
+        "C. Elvira and C. Herzet, “Safe rules for the identification of zeros in ",
+        "the solutions of the SLOPE problem.” arXiv, Oct. 04, 2022. ",
+        "doi: 10.48550/arXiv.2110.11784.",
+    ]
+    parameters = {
+        "accelerated": [True, False],
+    }
+
+    def set_objective(self, X, y, alphas, fit_intercept):
+        self.X, self.y, self.alphas = X, y, alphas
+        self.n_samples = len(y)
+        self.fit_intercept = fit_intercept
+
+        self.param = SlopeParameters()
+        self.param.accelerated = False
```

Comment:
Can we set it to self.accelerated directly here and remove the line in `run()` ?

Reply:
I agree, good point.